### ISSUE TITLE
esp8266 flash 

### ISSUE THREAD

**Issue by icefreesky** [herzig/bewegungsfelder#4]
Hi, need to flash 2 esp8266 with one "bewegungsfelder/bewegungsfelder_esp8266/esp8266_mpu6050/" firmware and it will be the main one, and the others "bewegungsfelder/bewegungsfelder_esp8266/esp8266_i2c_scan/" am I right? And do you remember the version of esp8266_SDK which was during the build?

**Comment by herzig** [comment]
No, you need to flash the ``esp8266_mpu6050`` to all of the ESP. Each with different ``SENSOR_ID``, and ``LOCAL_IP``.
Also make sure to change the SSID and Server settings to your network setup.
(https://github.com/herzig/bewegungsfelder/blob/develop/bewegungsfelder_esp8266/esp8266_mpu6050/user/user_main.c#L44)

The i2c scan code is just for testing and you don't actually need it for running the system.

Not sure about the exact SDK Version, I used this link: https://www.esp8266.com/viewtopic.php?t=820

**Comment by icefreesky** [comment]
Thank you, I used everything as you indicated, but no matter how I tried, when using unofficial sdk 2.0.1, two files 0x10000.bin and 0x00000.bin are created during assembly, but as I looked from the make file, 2 files 0x00000.bin and 0x40000 should be created. bin everything is collected without errors, but the name is different and when trying to write the error is that there is no 0x40000.
bin, if you rename the recording is in progress, but no matter how I try to record through eclipse or the flash download tool, an error constantly appears if you look through the terminal Fatal exception (0):
epc1 = 0x40210ccc, epc2 = 0x00000000, epc3 = 0x00000000, excvaddr = 0x00000000, depc = 0x00000000
in an endless loop, the esp 07 module I set all the parameters for recording in accordance with the characteristics, the module itself is serviceable when writing standard firmware, it works, maybe you still have bin files ready to check the problem in eclipse, or you still need esp 12e modules for the firmware to work

**Comment by herzig** [comment]
Sorry but I'm afraid I won't be able to help you with this at the moment :( . I don't have the hardware ready and also don't have the build toolchain setup to generate any ESP8266 files on any computer right now.
